src/components/views/context_menus/MessageContextMenu.tsx

```diff
@@ -122,7 +126,11 @@ export default class MessageContextMenu extends React.Component<IProps, IState>
         // HACK: Intentionally say we can't pin if the user doesn't want to use the functionality
         if (!SettingsStore.getValue("feature_pinning")) canPin = false;
 
-        this.setState({ canRedact, canPin });
+        const isLoc = isLocationEvent(this.props.mxEvent);
+        const canForward = !isLoc;
+        const canShare = !isLoc;
+
+        this.setState({ canRedact, canPin, canForward, canShare });
```

Comment:
merge went wrong, undid the review changes

Reply:
Ah, I'm really sorry for letting that slip through.

Reply:
Fixed in d17ed30aa1